reconcile/openshift_base.py

```diff
@@ -113,10 +112,9 @@ def init_specs_to_fetch(ri, oc_map,
                 msg = f"[{cluster}] cluster skipped."
```

Comment:
this is another situation we would want to handle in the same way IMO (following the previous comment). again, perfectly good to go in a future PR.

Reply:
Same question as above.

Reply:
The new solution should cover this case as well